app/scripts/lib/metrics.js

```diff
@@ -235,7 +235,7 @@ define([
       return filteredData;
     },
 
-    _send: function (data, url) {
+    _send: function (data, url, async) {
       var self = this;
       var payload = JSON.stringify(data);
 
```

Comment:
It looks like even if `async===false`, if the environment has `navigator.sendBeacon`, that will be used. Expected? If so, can you add a note about why that's OK?